### ISSUE TITLE
rrule documentation should explicitly say what "weekday constant" means

### ISSUE THREAD

**Issue by rmahajan14** [dateutil/dateutil#685]
At two places in the rrule documentation (http://dateutil.readthedocs.io/en/stable/rrule.html), we have:
"Must be one of the MO, TU, WE constants"
"one of the weekday constants (MO, TU, etc)"

Is it better to at least have one systematic reference to weekday constants - a table, maybe, towards the beginning of the documentation?

Monday(MO)
Tuesday(TU)
Wednesday(WE)
..

Its probably obvious, but never explicitly mentioned(in the rrule documentation) that we require the first two letters of the weekday to refer to that day.

**Comment by pganssle** [comment]
Yes this should be explained, possibly on a separate page. These are actually semi-shared between `rrule` and `relativedelta`. There are slight differences in the implementation, even though they have the same final level name. [rrule version](https://github.com/dateutil/dateutil/blob/master/dateutil/rrule.py#L66), [relativedelta version](https://github.com/dateutil/dateutil/blob/master/dateutil/relativedelta.py#L13). Probably they should be explained in both modules and some note made that they are slightly different from one another.
